core/src/main/scala/org/apache/spark/storage/BlockManagerMaster.scala

```diff
@@ -201,7 +201,7 @@ class BlockManagerMaster(
         Option[BlockStatus],
         Iterable[Option[BlockStatus]]]]
     val blockStatus = timeout.awaitResult(
-      Future.sequence[Option[BlockStatus], Iterable](futures)(cbf, ThreadUtils.sameThread))
+      Future.sequence(futures)(cbf, ThreadUtils.sameThread))
```

Comment:
immutable.Iterable seems to have been the right-er choice for CanBuildFrom here; doesn't require overriding the generic types. It didn't compile otherwise in 2.13.